rfcs/text/0009-telemetry.md

```diff
@@ -72,6 +72,6 @@ We do have somewhat detailed (albeit spotty) ways of getting some information ab
 
 There aren't really any alternatives — without directly sending information about usage, we only have proxy telemetry which are incomplete and often misleading (hello NPM downloads count).
 
-We could do opt-in instead of opt-out but that would mean most people wouldn't ever know to opt-in and our data collection would be probably useless as we'd only see a small % of user activity and never be sure what % it is of the real total.
+We could do opt-in instead of opt-out but that would mean most people wouldn't even know to opt-in and our data collection would be probably useless as we'd only see a small % of user activity and never be sure what % it is of the real total.
```

Comment:
I couldn't think of how `ever` would fit into the context so I replaced it with `even`. I'm not sure if this would be a desirable change tho.

Reply:
```suggestion
We could do opt-in instead of opt-out but that would mean most people wouldn't ever know to opt-in and our data collection would be probably useless as we'd only see a small % of user activity and never be sure what % it is of the real total.
```